assets/parquet/Dockerfile

```
@@ -0,0 +1,68 @@
+# FROM ghcr.io/y-scope/clp/clp-core-dependencies-x86-ubuntu-jammy:main
+FROM presto/prestissimo-dependency:ubuntu-22.04
+# FROM presto/prestissimo-dependency:latest
+# FROM prestodb/presto-native-dependency:latest
+
+# Install necessary packages
+RUN apt-get update;
+
+# Install necessary packages (alphabetized)
+RUN apt-get install -y \
```

Hadolint (2.12.0)

[info] Avoid additional packages by specifying `--no-install-recommends`

(DL3015)